.gitignore

```diff
@@ -9,3 +9,4 @@ gamer
 GAMER_CompareData
 GAMER_ExtractUniform
 GAMER_ExtractProfile
+*.bin
```

Comment:
Which new tools will generate `*.bin`?

Reply:
The interpolation tables used by the spectral interpolation algorithm are *.bin files. They are created by [this notebook](https://github.com/KunkelAlexander/my-gamer-fork/blob/2e038061d5dd4a4117a7a77448d838afcfb798db/tool/table_maker/GramFE/GramFE.ipynb)

Reply:
Got it.